crates/aiken-lang/src/parser/token.rs

```diff
@@ -176,6 +177,7 @@ impl fmt::Display for Token {
             Token::Test => "test",
             Token::Fail => "fail",
             Token::Validator => "validator",
+            Token::Via => "via",
```

Comment:
there's an inverse of this somewhere that always need to be hunted down and added. I believe it's a function called `str_to_keyword`